File: 8-kyu/calculate-bmi/swift/solution.swift
func bmi(_ weight: Int, _ height: Double) -> String {
    let bmi = Double(weight) / (height * height)
    return bmi <= 18.5 ? "Underweight"
        : bmi <= 25.0 ? "Normal"
            : bmi <= 30.0 ? "Overweight"
                : "Obese"
}

File: 8-kyu/calculate-bmi/javascript/solution.js
const bmi = (weight, height) => {
  const calculatedBmi = weight / (height * height);
  if (calculatedBmi <= 18.5) {
    return "Underweight";
  } else if (calculatedBmi <= 25.0) {
    return "Normal";
  } else if (calculatedBmi <= 30.0) {
    return "Overweight";
  }
  return "Obese";
};